### PR TITLE
Created profile

### PR DESCRIPTION
**Pull request by frontendherodk**
This is an optional description

@elezar Have a look

### CODE REVIEW

File: profiles/frontendherodk.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Henrik Andersen
+
+![](https://github.com/frontendherodk.png)
+
+#TODOs
+- [x] Register for Github4DeveloperTraining
```
Comment (elezar) [inline profiles/frontendherodk.md]: I think this should be "GitHub4Developers"

File: profiles/frontendherodk.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- [ ] Learn git(hub)
+- [ ] Master git(hub)
```
Comment (elezar) [inline profiles/frontendherodk.md:8]: - [ ] Profit :moneybag:

**Comment by frontendherodk** [comment]
@elezar Can you have a second look before I merge?

**Comment by elezar** [comment]
LGTM :+1: :ship: :shipit: :gun:

**Comment by brntbeer** [comment]
🚢

**Pull request merged**